src/admin/index.html

```diff
@@ -3,9 +3,8 @@
   <head>
     <title>ADMIN - USER PORTAL</title>
   </head>
-  <body style="margin:0;padding:0">
-    <div id="root">
-    </div>
+  <body style="margin: 0; padding: 0">
+    <div id="root"></div>

```

Comment:
Not sure if you manually did this change, but if you did you might as well have made it `<div id="root" />` :)

Reply:
@LindvedKrvang `<div />` is not valid HTML. It might be that angular, react and other frameworks allow this notation – but that is just syntactic sugar over the HTML standard.